codebase/base/src.lib/math/rmath.1.8/include/rmath.h

```diff
@@ -34,23 +38,28 @@
 #ifndef _RMATH_H
 #define _RMATH_H
 
-#ifndef _COMPLEX
+/*Need to use _COMPLEX_H to not
+ * override the complex.h definition*/
+#ifndef _COMPLEX_H
 struct complex {
     double x,y;
 };
 #define cabs(a) sqrt((a.x*a.x)+(a.y*a.y))
 #endif
 
-#define C 2.997e+8
+/*Speed of Light in m/s*/
+#define C 2.99792458e+8
 #define LN_TO_LOG 4.342944819
 
 #ifndef HUGE_VAL
 #define HUGE_VAL 1.0e36                /* default value of HUGE */
 #endif
  
-#ifndef PI
```

Comment:
Maybe I'm being daft, but if one were to include both `math.h` and `rmath.h` (as is done in codebase/base/src.lib/graphic/fbuffer.1.19/src/matrix.c) then removing the `ifndef` here overwrites the existing `PI` definition? Maybe that's ok, just pointing it out.

Reply:
So math.h actually reference Pi as `M_PI` so this library wouldn't affect rmath.h `PI` constant. I hope this would prevent people from using math.h `PI` for RST's consistency sake.
 
https://www.gnu.org/software/libc/manual/html_node/Mathematical-Constants.html
https://www.tutorialspoint.com/c_standard_library/math_h.htm

Granted I left `ifndef` on `HUGE_VAL` which I think is also defined in math.h. 

I had to make `ifndef` around `complex` just because that is a complicated other issues too with RST defining predefined data structures.

I was thinking we want `rmath.h` to overwrite to ensure consistency in other predefined libraries this may help avoid weird behaviors too. However, the external libraries we include in RST is hard to say, but I think that is the authors of those libraries problem ... or we need to come up with a better strategy but that is out of scope for this PR.

Reply:
Ha, totally called it, I was being daft. I got `PI` and `M_PI` confused...